### PR TITLE
feat: upgrade opentelemtry

### PR DESCRIPTION
**Pull request by btpemercier**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by btpemercier** [comment]
@microsoft-github-policy-service agree

**Comment by jsquire** [comment]
@lmolkova, @JoshLove-msft:  Would you please review?

**Review by vishweshbankwar — APPROVED** [review]
LGTM (Approving the OTel part)

File: sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag>Insights_2018-03-01;Insights_2018-11-27-preview;insights_2019-03-01;insights_2017-04-01;insights_2015-04-01;insights_2016-03-01;insights_2017-05-01-preview;insights_2017-11-01-preview;insights_2018-01-01;insights_2017-12-01-preview;insights_2018-04-16;insights_2019-10-17-preview;insights_2021-04-01;insights_2021-09-01;insights_2022-02-01;insights_2022-06-01</AzureApiTag>
+    <AzureApiTag>Insights_2022-06-01;Insights_2020-10-01;Insights_2015-04-01;Insights_2016-03-01;Insights_2019-03-01;Insights_2021-09-01-preview;Insights_2017-05-01-preview;Insights_2018-03-01;Insights_2018-01-01;Insights_2019-10-17-preview;Insights_2018-04-16;Insights_2018-11-27-preview;insights_2017-12-01-preview;</AzureApiTag>
```
Comment (vishweshbankwar) [inline sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props:4]: This looks like unrelated change. Is this intended?

Comment (acesyde) [reply sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props:4]: Hi @vishweshbankwar,

I have no idea.  I just change the dependency value for OpenTelemetry, launch the build and the tests like it's describe in the contributing et voilà

Comment (vishweshbankwar) [reply sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props:4]: Ok - please revert this change and we can merge then. Thanks!

Comment (btpemercier) [reply sdk/monitor/Microsoft.Azure.Management.Monitor/AzSdk.RP.props]: @vishweshbankwar done 👍

**Review by TimothyMothra — APPROVED** [review]
